modules/runtime/nuxeo-launcher-commons/src/test/java/org/nuxeo/launcher/config/ConfigurationGeneratorTest.java

```diff
@@ -561,22 +527,4 @@ protected boolean isTemplateIncluded(String template) {
         return configGenerator.getIncludedTemplates().stream().map(File::getName).anyMatch(isEqual(template));
     }
 
-    /**
-     * Checks environment variable replacement within templates (NXP-29392).
-     *
-     * @since 11.2
-     */
-    @Test
-    public void testEnvironmentVariableInNuxeoDefaults() {
```

Comment:
is this feature covered in another test?

Reply:
Yes, it has been moved [there](https://github.com/nuxeo/nuxeo/pull/4603/files#diff-f201073b7409397d423bfc3eeaee09a2c409e7fe392e2638788d14e34b5eeaecR67-R85).